roster/src/Roster.Infrastructure/Storage/LockStatementProvider.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using MassTransit;
+using MassTransit.EntityFrameworkCoreIntegration;
+using MassTransit.Metadata;
+using MassTransit.Saga;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Roster.Infrastructure.Storage
+{
+    public class LockStatementProvider : ILockStatementProvider
```

Comment:
If I remember correctly this is required by MassTransit having issues with EF Core 6.0 and is the suggested fix until they address the compatibility issues (https://github.com/MassTransit/MassTransit/issues/2980). Would be good to add a reference in a comment so we remember why this is here